packages/app/obojobo-repository/shared/components/collection-module-list-item.js

```diff
@@ -0,0 +1,20 @@
+require('./collection-module-list-item.scss')
+
+const React = require('react')
+const ModuleImage = require('./module-image')
+
+const CollectionModuleListItem = props => {
+	const classNames = [
+		'module-list-item',
+		props.alreadyInCollection ? 'is-already-in-collection' : 'is-not-already-in-collection'
```

Comment:
We have an is-or-not.js helper function to create these strings

Reply:
Trying to use that helper function in any of the repository components throws up a lot of module import-related errors that are not even remotely close to worth dealing with to replace a single line of code.